django_descope/models.py

```diff
@@ -3,9 +3,8 @@
 from descope import DescopeClient
 from django.contrib.auth import models as auth_models
 from django.core.cache import cache
-from django.utils.functional import cached_property
 
-from .settings import IS_STAFF_ROLE, IS_SUPERUSER_ROLE, PROJECT_ID
+from .settings import IS_STAFF_ROLE, IS_SUPERUSER_ROLE, MANAGEMENT_KEY, PROJECT_ID
```

Comment:
should this be `DESCOPE_MANAGEMENT_KEY` and `DESCOPE_PROJECT_ID`?

Reply:
it seems to work regardless of this actually, but I have no idea why :thinking:

Reply:
This actually comes from the app settings and not the project settings

See django_descope/settings.py